src/common/io.go

```diff
@@ -0,0 +1,16 @@
+package common
+
+import "io"
+
+func WriteAll(writer io.Writer, data []byte) (n int, err error) {
```

Comment:
Where do you use this?

Reply:
Inside Vehicle.SendOrder to ensure every byte is written